arch/csr/H/vsie.yaml

```diff
@@ -0,0 +1,58 @@
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: vsie
+address: 0x204
+virtual_address: 0x144
+long_name: Virtual Supervisor Interrupt Enable
+description: |
+  The vsie register is a VSXLEN-bit read/write register that is VS-mode’s version of
+  supervisor CSR sie. When V=1, vsie substitutes for the usual sie, so instructions that
+  normally read or modify sie actually access vsie instead. However, interrupts directed to
+  HS-level continue to be indicated in the HS-level sip register, not in vsip, when V=1.
+
+  The standard portion (bits 15:0) of vsie is formatted as follows:
+  15 14 13 12 10 9 8 6 5 4 2 1 0
+  0 LCOFIE 0 SEIE 0 STIE 0 SSIE 0
+  2 1 3 1 3 1 3 1 1
+
```

Comment:
This is obviously not formatted well. Delete it. The fields information below covers the same information.